roofit/xroofit/inc/RooFit/xRooFit/xRooNode.h

```diff
@@ -114,7 +114,7 @@ class xRooNode : public TNamed, public std::vector<std::shared_ptr<xRooNode>> {
    }
    xRooNode(double value);
 
-   ~xRooNode() override;
+   virtual ~xRooNode();
```

Comment:
Please add back the `override` keyword to mark that this destructor is overriding, and the redundant `virtual` keyword should be removed (if the base class, in this case `TNamed` has already a virtual keyword, you don't need it here.